### ISSUE TITLE
CropTransformation makes the image bigger

### ISSUE THREAD

**Issue by eddieyanez** [microspaze/FFImageLoading.Maui#32]
### Description

Running CropTransform in iOS code-behind makes the image bigger even when using zoom factor 1, and offsets 0.

```
var jpgStream = await imageService
                        .LoadFile(Path)
                        .Transform(new CropTransformation(1, 0, 0))
                        .AsJPGStreamAsync(imageService, 70);
```

Without CropTransformation:

Result width: 2160 pixels
Result height: 1434 pixels
File size in bytes: 12,389,760

With CropTransformation:

Result width: 4302 pixels
Result height: 4302 pixels
File size in bytes: 74,028,816

### Steps to Reproduce

As above.

### Expected Behavior

No file size change.

### Actual Behavior

Image somehow grows in size.

### Basic Information

- Version with issue: 1.2.1
- Last known good version: N/A
- Platform: iOS, I've not checked on Android or other platforms

### Screenshots

N/A

### Reproduction Link / Code

N/A

**Comment by microspaze** [comment]
Fixed in the latest version 1.2.2 https://www.nuget.org/packages/FFImageLoading.Maui/1.2.2

Thanks for reporting this issue. Please help to verify the issue fixing when you are free.

**Comment by microspaze** [comment]
@eddieyanez Sorry for not reverting the test code change in CropTransformation. Fixed version 1.2.3 has been uploaded to Nuget.  https://www.nuget.org/packages/FFImageLoading.Maui/1.2.3
